### ISSUE TITLE
addon-a11y detects addon-info violations

### ISSUE THREAD

**Issue by mrsufgi** [storybooks/storybook#5680]
**Describe the bug**

that's pretty funny but when I use both withInfo and withA11Y decorators globally together (in v5-rc.3)
the accessibility addon detects: 

``Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds``

with the problematic class - .info__show-button  which belongs to the info button :D 
 
  
**To Reproduce**
add both plugins globally 
(order doesn't matter) 

```js
addDecorator(withA11Y);
addDecorator(withInfo); 
```

check a11y panel

**Expected behavior**
0 issues regarding storybook addons 

**System:**
 - OS:  MacOS
 - Device: MBR
 - Browser: Chrome
 - Framework: React
 - Addons: a11y, info
 - Version: 5.0.0-rc.3

**Comment by shilman** [comment]
We're going to deprecate `addon-info` soon and move it into the addons panel, which should address this problem: https://github.com/storybooks/storybook/issues/1147

**Comment by mrsufgi** [comment]
so until then, I'll drop addon-info from my storybook :) (i guess it has issues with storyshots too)

**Comment by shilman** [comment]
Sorry for the inconvenience. addon-info has been a thorn in our side for a long time now -- I'm really excited to get something better in place as soon as v5 has been released!
